code-challenges.js

```diff
@@ -69,7 +122,29 @@ var vowelTester2 = "throw"
 var websites = ["codepen.io", "codecademy.com", "coursera.org", "codepen.io", "udemy.com", "pluralsight.com", "udacity.com", "sitepoint.com"]
 // Expected output: "codepen.io" "codepen.io"
 
+// Create a function that takes in the array of websites
+const newWebsite = (arr) => {
+// find a way to look through the array and find the end since .io is only at the end 
+    let ios = arr.filter(value => value.endsWith(".io")) 
+// return only the onlys that are .io
+    return ios 
+    //return ios.join(" ") if you wanted to make them into a string
+}
```

Comment:
This is great! Since the expected output is supposed to be just two strings, I'd definitely be sure to include that .join. 

Additionally, the way this is written is TOTALLY fine. Just want to provide you another way it could be written to be a little more concise! 

`const newWebsite = (arr) => {`
     ` return arr.filter(value => value.endsWith(".io")).join(" ") `
 ` }`

Since we're already passing a variable into the function, there is not really a need to create a brand new one! We already essentially have `websites.filter(...` so the idea of creating a new variable named 'ios' is basically saying `websites = websites.filter(...`. We can just return the arr.filter right off the bat, without having to create the new variable and return that variable again right after it. Additionally we can string our methods on to it, so long as it follows the order of operation! 

Again I want to mention, the way you wrote it is TOTALLY fine, but I just wanted to provide you with something to consider and think about in the future! :)